### ISSUE TITLE
Model accuracy

### ISSUE THREAD

**Issue by Ulduzpp** [jpcurbelo/human-body-reshape-DL-paper#9]
Hi @jpcurbelo 
Thanks for sharing your great work in github, it is splendid, I just wanted to ask you about the accuracy of model you provided, and also do we need to retrain the model or finetune it?

**Comment by jpcurbelo** [comment]
Hi @Ulduzpp. Thanks for your interest in our work! We did run some experiments to assess accuracy (see plot). You can find the details [in our paper](https://link.springer.com/article/10.1007/s11042-023-17947-6) (also see the plot). In case you can not access the paper, please, share your email and I will send you a copy. 

Also, see that [the output file](https://github.com/jpcurbelo/human-body-reshape-DL-paper/blob/master/data/output_files/output_data_avatar_male_fromImg.csv) includes the relative percentage deviation (δ%). This is the measurements taken from the generated avatar compared with the predicted/imputed ones.

Please, let me know if you have further questions, Happy to help.
![image](https://github.com/user-attachments/assets/105e8d87-660b-408f-b604-21232eb85d31)

**Comment by jpcurbelo** [comment]
About retraining the model. [The script to train the extractor model](https://github.com/jpcurbelo/human-body-reshape-DL-paper/blob/master/src/extractor/extractor_model_training.py) can be adapted and used for different configurations. You might want to play around to include different measurements to be used as inputs and labels as well.

**Comment by Ulduzpp** [comment]
Hi @jpcurbelo thanks for your comprehensive response and about the paper this is my email: u.pakpoor@gmail.com
 I would like to have a copy of your work to read

**Comment by Ulduzpp** [comment]
also I mostly am curious about the accuracy of body measurements I saw that you have written it has 97% accuracy for body measurements but I would like to know if it consists children as well or not besides are there especial rules that we should consider while giving image inputs to the model? cause I tested it but the weight and chest girth were too different from reality that is why I ask that maybe it is because of my input images.

**Comment by jpcurbelo** [comment]
Hi @Ulduzpp. I just sent the paper by email. 

About results and input files:

1- The NN to extract the measurements was trained on [ANSUR II data](https://www.openlab.psu.edu/ansur2/) that only contains adult subject. Therefore, I don't expect the model to perform well on children data. It would be interesting though to re-train the model on a children dataset to see what happens but I guess it won't be easy to find images available.

2- For the input images, please, [see this comment]( https://github.com/jpcurbelo/human-body-reshape-DL-paper/issues/8#issuecomment-2387043943). I would like to see the silhouettes extracted during from experiments if you don't mind to share the files - just the silhouettes, not the original photos of the person. Thanks.

**Comment by Ulduzpp** [comment]
Thank you @jpcurbelo for your email and the detailed comments, I have received the paper.
also about the my experiment this is the silhouettes I have got, besides the height is 171 and male 
[silhouettes.zip](https://github.com/user-attachments/files/18219694/silhouettes.zip)

**Comment by Daheer** [comment]
Hello @jpcurbelo, I am also very interested in reading the paper, this is my email: suhayrid6@gmail.com

Thank you
